src/validator/Tests.java

```diff
@@ -0,0 +1,55 @@
+package validator;
+
+import static org.junit.Assert.*;
+
+import java.io.BufferedReader;
+import java.io.InputStreamReader;
+import java.util.ArrayList;
+
+import org.junit.Test;
+
+public class Tests {
+	private static CreditCardVendorsReader reader = new CreditCardVendorsReader();
```

Comment:
why static members? good practice is to avoid static whenever this is possible you could create setup function which will create validator object without reading a file. Generally unit test should avoid reaching out to external resources